apps/editors/tests/test_helpers.py

```diff
@@ -229,11 +229,6 @@ def test_no_version(self):
                                       version=None)
         eq_(helper.review_type, 'pending')
 
-    def test_review_files(self):
-        for status in REVIEW_FILES_STATUSES:
```

Comment:
Need to get rid of this constant on line 32 and merge its values with the `REVIEW_ADDON_STATUSES` constant on the line above it. `test_nominated_review_time_set` and `test_preliminary_review_time_set` will also need to be merged, somehow.